diff --git a/src/Window.cpp b/src/Window.cpp
index 5a50713b2d..f41099dff4 100644
--- a/src/Window.cpp
+++ b/src/Window.cpp
@@ -1066,7 +1066,7 @@ bool CWindow::canBeTorn() {
 
 bool CWindow::shouldSendFullscreenState() {
     const auto MODE = g_pCompositor->getWorkspaceByID(m_iWorkspaceID)->m_efFullscreenMode;
-    return m_bFakeFullscreenState || (m_bIsFullscreen && (MODE == FULLSCREEN_FULL));
+    return m_bFakeFullscreenState; //|| (m_bIsFullscreen && (MODE == FULLSCREEN_FULL));
 }
 
 void CWindow::setSuspended(bool suspend) {
